File: Dart/1037.dart
import 'dart:io';

void main() {
  double x = double.parse(stdin.readLineSync());
  if (x >= 0 && x <= 25) {
    print("Intervalo [0,25]");
  } else if (x > 25 && x <= 50) {
    print("Intervalo (25,50]");
  } else if (x > 50 && x <= 75) {
    print("Intervalo (50,75]");
  } else if (x > 75 && x <= 100) {
    print("Intervalo (75,100]");
  } else {
    print("Fora de intervalo");
  }
}

File: PHP/1037.php
<?php
$x = rtrim(fgets(STDIN));
if ($x >= 0 && $x <= 25) {
    echo "Intervalo [0,25]" . PHP_EOL;
} else if ($x > 25 && $x <= 50) {
    echo "Intervalo (25,50]" . PHP_EOL;
} else if ($x > 50 && $x <= 75) {
    echo "Intervalo (50,75]" . PHP_EOL;
} else if ($x > 75 && $x <= 100) {
    echo "Intervalo (75,100]" . PHP_EOL;
} else {
    echo "Fora de intervalo" . PHP_EOL;
}
